magento/pwa-studio
# Search Result Counts Match

## Description

There are two places we show the count of a search result: in the autocomplete dropdown and on the search page itself. Unfortunately, these two places were calculating their values differently and so sometimes did not match.

This PR:
  * Updates the way the autocomplete dropdown was calculating its value
  * Updates the search results page to add pagination so the number of items matches the number of results indicated

## Related Issue





Closes JIRA-97.

## Acceptance 



### Verification Stakeholders

### Specification



### Verification Steps


1. Perform a search for "Tops Dresses"
1. Note how many items the autocomplete dropdown says have been found
1. Press `<Enter>` to execute the search and continue to the search page
1. Note how many items the search page says have been found
1. See that the two numbers match

## Screenshots / Screen Captures (if appropriate)

### Autocomplete

![Screen Shot 2019-12-18 at 11 03 07 AM](https://user-images.githubusercontent.com/13182778/71107145-2fd45400-2186-11ea-8bee-fb6c8bbbd153.png)

### Search Page

![Screen Shot 2019-12-18 at 11 03 16 AM](https://user-images.githubusercontent.com/13182778/71107163-3662cb80-2186-11ea-89b7-0ab24d829165.png)

## Checklist

* I have updated the documentation accordingly, if necessary.
* I have added tests to cover my changes, if necessary.
